types/webidl.d.ts

```diff
@@ -55,9 +55,7 @@ interface WebidlUtil {
     V: unknown,
     bitLength: number,
     signedness: 'signed' | 'unsigned',
-    opts?: ConvertToIntOpts,
-    prefix: string,
-    argument: string
+    opts?: ConvertToIntOpts
```

Comment:
tsc reported that after optional paramaters there can not be mandatory parameters.

Compared with implementation and the following arguments are not existing.